### DISCUSSION TITLE
What about hypervisor on m1 Ipad with new MacDirtyCow exploit?

### DISCUSSION THREAD

**Discussion by Maxfusion26** [utmapp/UTM#4853] [UTM Help (iOS)]
What about hypervisor on m1 Ipad with new MacDirtyCow exploit?

**Comment by TheToto** [comment]


**Reply by Maxfusion26** [reply]
> That is not a iPadOS exploit.

The MacDirtyCow bug, CVE-2022-46689, for iPhones and iPads running iOS or iPadOS 15.x-16.1.2.

**Reply by TheToto** [reply]
Thank you, my bad

**Comment by Andy312432** [comment]
Not directly possible I think🤔
It can replace the system files, but not signing the code(or entitlements)

**Comment by MotorBottle** [comment]
According to OSY’s twitter, seems like it’s not possible

**Reply by MotorBottle** [reply]
But I would keep my iPad staying on iOS 16.1.1 after I all, cuz JB for iOS 16 might most likely to be deployed on these earlier versions.
